File: packages/algo-py/solutions/algo_04_test.py
import unittest
from solutions import min_path_sum


class TestAlgo_04(unittest.TestCase):
    def test(self):
        input = [
                [1,3,1],
                [1,5,1],
                [4,2,1],
                ]
        self.assertEqual(min_path_sum(input), 7)


if __name__ == '__main__':
    unittest.main()

File: packages/algo-go/solutions/algo-04_test.go
package solutions

import "testing"

func TestMinPathSum(t *testing.T) {
	input := [][]int{
		{1, 3, 1},
		{1, 5, 1},
		{4, 2, 1},
	}

	if MinPathSum(input) != 7 {
		t.Error("Failed")
	}
}
